example/image-classification/benchmark_score.py

```diff
@@ -21,26 +21,47 @@
 from common import find_mxnet
 from common.util import get_gpus
 import mxnet as mx
+import mxnet.gluon.model_zoo.vision as models
 from importlib import import_module
 import logging
+import argparse
 import time
 import numpy as np
 logging.basicConfig(level=logging.DEBUG)
 
+parser = argparse.ArgumentParser(description='SymbolAPI-based CNN perf')
+parser.add_argument('--network', type=str, default='all', 
+                                 choices=['all', 'alexnet', 'vgg-16', 'resnetv1-50', 'resnet-50',
+                                          'resnet-152', 'inception-bn', 'inception-v3', 
+                                          'inception-v4', 'inception-resnet-v2', 'mobilenet',
+                                          'densenet121', 'squeezenet1.1'])
+parser.add_argument('--batch-size', type=int, default=0)
```

Comment:
same as above, default should be 32?

Reply:
default = 0 means run all batchsize in list [1, 2, 4, 8, 16, 32]. I don't want to break the default action of this script and users can set to a specific numbers.